apps/docs/content/guides/platform/manage-your-usage/compute.mdx

```diff
@@ -0,0 +1,117 @@
+</Admonition>
+
+## Billing examples
+### One project
+The project runs on the same Compute size throughout the entire billing cycle.
+
+| Line Item                     | Hours | Costs   |
+|-------------------------------|-------|---------|
+| Pro Plan                      | -     | $25     |
+| Compute Hours Micro Project 1 | 744   | $10     |
+| **Subtotal**                  |       | **$35** |
+| Compute Credits               |       | -$10    |
+| **Total**                     |       | **$25** |
+
+### Multiple projects
+All projects run on the same Compute size throughout the entire billing cycle.
+
+| Line Item                     | Hours | Costs   |
+|-------------------------------|-------|---------|
+| Pro Plan                      | -     | $25     |
+| Compute Hours Micro Project 1 | 744   | $10     |
+| Compute Hours Micro Project 2 | 744   | $10     |
+| Compute Hours Micro Project 3 | 744   | $10     |
+| **Subtotal**                  |       | **$55** |
+| Compute Credits               |       | -$10    |
+| **Total**                     |       | **$45** |
+
+### One project on different Compute sizes
+The project's Compute size changes throughout the billing cycle.
+
+| Line Item                     | Hours | Costs   |
+|-------------------------------|-------|---------|
+| Pro Plan                      | -     | $25     |
+| Compute Hours Micro Project 1 | 233   | $3      |
+| Compute Hours Small Project 1 | 511   | $11     |
+| **Subtotal**                  |       | **$39** |
+| Compute Credits               |       | -$10    |
+| **Total**                     |       | **$29** |
+
+## View Compute usage
+You can view Compute usage on the [organization's usage page](https://supabase.com/dashboard/org/_/usage). The page shows the usage of all projects by default. To view the usage for a specific project, select it from the dropdown. You can also select a different time period.
+<Image
+  alt="Usage page navigation bar"
+  src={{
+    light: '/docs/img/guides/platform/usage-navbar--light.png',
+    dark: '/docs/img/guides/platform/usage-navbar--dark.png',
+  }}
+  zoomable
+/>
+
+In the Compute Hours section, you can see how many hours of a specific Compute size your projects have used during the selected time period. Hover over a specific date for a daily breakdown.
+<Image
+  alt="Usage page Compute Hours section"
+  src={{
+    light: '/docs/img/guides/platform/usage-compute--light.png',
+    dark: '/docs/img/guides/platform/usage-compute--dark.png',
+  }}
+  zoomable
+/>
+
+## Optimize Compute usage
+- load test your application in staging to understand your Compute requirements
+- start out on a smaller Compute size, [create a report](https://supabase.com/dashboard/project/_/reports) in the Dashboard to monitor your CPU and memory utilization, and upgrade the Compute size as needed
```

Comment:
Is this supposed to be the section on how to reduce usage?

If yes, let's add something along the lines of
- Delete projects or transfer a paid plan project using self-serve project transfers to reduce compute usage

Reply:
Done